src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -237,6 +240,13 @@ public int instanceMain(final String[] argv) {
         if (System.getProperty("ga4gh.client_secrets") == null) {
           System.setProperty("ga4gh.client_secrets", GA4GH_CLIENT_SECRETS);
         }
+
+        if (System.getProperty("google_project_requester_pays") == null && REQUESTER_PAYS_PROJECT != null) {
```

Comment:
There's a way to get the default project from the environment I think.  I don't know if you want to get that by default?

Reply:
does GATK do that?